lib/ansible/modules/network/nxos/nxos_gir.py

```diff
@@ -161,14 +166,7 @@
 
 def execute_show_command(command, module, command_type='cli_show_ascii'):
     cmds = [command]
-    device_info = get_capabilities(module)
-    network_api = device_info.get('network_api', 'nxapi')
-
-    if network_api == 'cliconf':
-        body = run_commands(module, cmds)
-    elif network_api == 'nxapi':
-        body = run_commands(module, cmds)
-
+    body = run_commands(module, cmds)
```

Comment:
@saichint Is `execute_show_command` even required? We can directly use `run_commands` since this method is calling `run_commands` only.